src/utils/__tests__/index.ts

```diff
@@ -1121,7 +1124,10 @@ describe('signerToSignatory and signatoryToSigner', () => {
       AccountKey: polkadotMockUtils.createMockAccountKey(fakeResult.value),
     });
 
-    encodedAddressStub.withArgs(someAccountKey).returns(someAccountKey);
+    sinon
```

Comment:
Again, this shouldn't return the same value that is passed to it

Reply:
I fixed it. weird